k2/csrc/log.cuh

```diff
@@ -171,7 +194,7 @@ class Voidifier {
 #define K2_LOG(x) \
   ::k2::internal::Logger(__FILE__, __func__, __LINE__, ::k2::internal::x)
 
-#define K2_CHECK_CUDA_ERROR(x) \
+#define K2_CHECK_CUDA_ERROR(x...)  \
```

Comment:
could you explain what the meaning of the three dots `...` is ?

Reply:
Occasionally, the arg is a call that may contain spaces. The `...` make the whole thing between brackets as one arg. I checked current codes. Without it is fine. And the `K2_CUDA_SAFE_CALL` is already presented. It's fine enough, restored it.